scheduler/src/cook/kubernetes/api.clj

```diff
@@ -82,11 +82,14 @@
   (when-not (synthetic-pod? pod-name)
     pod-name))
 
-(defn assoc-uuid
-  "Return a map with :job-uuid or :instance-uuid based on pod-name"
+(defn assoc-uuids
+  "Returns event-map with added fields :job-uuid (based on pod-name) and :instance-uuid (if pod is not synthetic)"
   [event-map pod-name]
   (let [instance-uuid (pod-name->instance-uuid pod-name)
-        job-uuid (pod-name->job-uuid pod-name)]
+        synthetic-pod-job-uuid (synthetic-pod-name->job-uuid pod-name)
+        job-uuid (if synthetic-pod-job-uuid
+                   synthetic-pod-job-uuid
+                   (tools/instance-uuid->job-uuid-cache-lookup instance-uuid))]
```

Comment:
Can be simplified to:

```suggestion
        job-uuid (or synthetic-pod-job-uuid
                   (tools/instance-uuid->job-uuid-cache-lookup instance-uuid))]
```

Reply:
Yeah I'll definitely use this. I thought Clojure would have a sleeker way to do it than my solution but I didn't know `or` worked like this, very interesting.